dali/python/backend_impl.cc

```diff
@@ -299,6 +327,27 @@ void ExposeTensorList(py::module &m) {
       Parameters
       ----------
       )code")
+      .def("__getitem__",
+        [](TensorList<CPUBackend> &t, Index id) -> std::unique_ptr<Tensor<CPUBackend>> {
+          return TensorListGetItemImpl(t, id);
+        },
+      R"code(
+      Returns a tensor at given position in the list.
+
+      Parameters
+      ----------
+      )code",
+      py::keep_alive<0, 1>())
```

Comment:
Why `py::keep_alive<0, 1>()` and the other way around below?
Shouldn't it be `py::keep_alive<1, 0>()` in both cases?